apps/projects/app/components/Panel/NewIssue/NewIssue.js

```diff
@@ -103,66 +112,125 @@ class NewIssue extends React.PureComponent {
+        labels: { totalCount: 0, edges: Array(0) },
+        milestone: null,
+        state: 'OPEN',
+        url: null,
+        createdAt: new Date()
+      }
+      newIssueData.issueId = newIssueData.id
+      const hashedIssueData = await ipfsAdd(newIssueData)
+      api.setIssue(repoHexIds[selectedProject - 1], newIssueData.number, hashedIssueData).toPromise()
+    }
+
     // TODO: refetch Issues list after mutation
+    const [{ loading, error }] = this.props.graphqlMutation
+    if (this.props.scope && selectedProject > 0 && !reposManaged[selectedProject - 1].decoupled) return <AuthorizeGitHub />
+
+    if (loading) return <Creating />
+
+    if (error) return <div css={`margin-top: ${3 * GU}px`}>Error</div>
 
     return (
-      <div css={`margin: ${2 * GU}px 0`}>
-        <Mutation
-          mutation={NEW_ISSUE}
-          variables={{ title, description, id }}
-          onError={console.error}
-        >
-          {(newIssue, result) => {
-            const { data, loading, error, called } = result
-            if (!called) {
-              return (
-                <Form
-                  onSubmit={newIssue}
-                  submitText="Submit Issue"
-                  submitDisabled={this.canSubmit()}
-                >
-                  <Field label="Project">
-                    <DropDown
-                      items={items}
-                      selected={selectedProject}
-                      onChange={projectChange}
-                      wide
-                      required
-                    />
-                  </Field>
-                  <Field label="Title">
-                    <TextInput onChange={titleChange} required wide />
-                  </Field>
-                  <Field label="Description">
-                    <TextInput.Multiline
-                      rows={3}
-                      style={{
-                        resize: 'none',
-                        height: 'auto',
-                        paddingTop: '5px',
-                        paddingBottom: '5px',
-                      }}
-                      onChange={descriptionChange}
-                      wide
-                    />
-                  </Field>
-                </Form>
-              )
-            } // end if(!called)
-            if (loading) {
-              return <Creating />
-            }
-            if (error) {
-              return <div>Error</div>
-            }
-
-            const { createIssue } = data
-            if (createIssue) {
-              this.props.closePanel()
-            }
-            return null
-          }}
-        </Mutation>
+      <div css={`margin: ${3 * GU}px 0`}>
+
+        <Field label="Project">
+          <DropDown
+            items={items}
+            selected={selectedProject}
+            onChange={projectChange}
+            wide
+            required
+          />
+        </Field>
+        {selectedProject > 0 && (reposManaged[selectedProject - 1].decoupled ? (
```

Comment:
No need to change right now, but just FYI we can use the dropdown placeholder to avoid these index modification hacks that are error-prone:
https://github.com/aragon/aragon-ui/tree/master/src/components/DropDown#placeholder